beacon_node/http_api/src/lib.rs

```diff
@@ -4652,8 +4652,10 @@ pub fn serve<T: BeaconChainTypes>(
                         post_beacon_blocks_ssz
                             .uor(post_beacon_blocks_v2_ssz)
                             .uor(post_beacon_blinded_blocks_ssz)
-                            .uor(post_beacon_blinded_blocks_v2_ssz),
+                            .uor(post_beacon_blinded_blocks_v2_ssz)
+                            .boxed(),
                     )
+                    .boxed()
```

Comment:
I feel like this is the critical one, although I feel like the handler/filter shouldn't be too massive at this point as it is basically:

```rust
post().and(header_exact.and(boxed_handler))
```

which would have a type something like:

```rust
And<Post, And<HeaderExact, Boxed<Filter<..>>>
```

The size of this object on the stack does not seem to me like it should be large. An `And` is only as large as its two elements added together:

https://github.com/seanmonstar/warp/blob/7b07043cee0ca24e912155db4e8f6d9ab7c049ed/src/filter/and.rs#L13-L16

The fact we've already boxed most of the handler should help here, I would think.

Reply:
Thinking it may be more related to the stack size of the futures created by the filters when they execute:

https://github.com/seanmonstar/warp/blob/7b07043cee0ca24e912155db4e8f6d9ab7c049ed/src/filter/and.rs#L38-L50

AFAICT both request and response are boxed anyway